src/app.service.js

```diff
@@ -5,7 +5,20 @@ import { HttpError, HttpStatus } from "./utils/http";
 
 const EXCLUDED_TAGS_REGEX =
   /^(button|img|nav|aside|footer|audio|canvas|embed|iframe|map|area|noscript|object|option|optgroup|picture|progress|script|select|source|style|svg|meta)$/i;
+const EXCLUDED_CLASS_NAMES_REGEX =
+  /(button|btn|nav|footer|summary|author|comment|sns|share|card|activity|relate|another|reply|sr-only|post-switch|loof|revenue|act|tag|direction|og|notice)/i;
+const EXCLUDED_ID_REGEX = /(paging|player)/i;
 const MAIN_CONTENT_TAGS_REGEX = /^(main|article|section)$/i;
+const MAIN_CONTENT_CSS_REGEX =
+  /^(tt_article_useless_p_margin|area_view|contents_style|contents_style|article_skin)$/i;
```

Comment:
```suggestion
const MAIN_CONTENT_CSS_REGEX =
  /(tt_article_useless_p_margin|area_view|contents_style|contents_style|article_skin)/i;
```
정규표현식의 ^는 문자열의 시작을 나타내고, $는 문자열의 끝을 의미합니다.
즉, 현재 작성된 정규표현식은 전체 문자열이 괄호 안의 항목 중 하나와 정확히 일치해야 함을 나타내고 있습니다.

물론 class에 괄호 안의 항목 중 하나만 사용한 경우에는 문제가 없겠지만 다른 클래스명이 함께 사용된다면 의도와 다르게 본문으로 판별하지 못할 것 같습니다.
ex) `<div class="tt_article_useless_p_margin foo"> ... </div>`